content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -59,11 +59,14 @@ generator client {
 
 ## Multiple extensions
 
-Each time that you call `$extends`, you create an _extended client_. This extended client is a forked variant of the standard Prisma Client. The original client does not change.
+When you use multiple extensions, the following statements apply to the associated extended clients:
 
-Every extended client operates independently, in an isolated instance. It cannot conflict with other extended clients or the original client.
+- Every extended client operates independently in an isolated instance
+- Extended clients cannot conflict with each other, or with the original client
+- All extended clients communicate with the same [Prisma query engine](/concepts/components/prisma-engines/query-engine)
+- All extended clients share the same comnnection pool
```

Comment:
I think line 62-67 actually applies outside of "Multiple extensions" as well - with just 1 standard and 1 extended Client all these statements are already true.

Reply:
I've moved this into the intro section